lib/state.js

```diff
@@ -657,6 +733,14 @@ class State {
     assert(typeof conditions === 'object');
     return this._listTable('spotrequests', conditions, client);
   }
+   
+  /**
+   * Get a list of AMIs and their usage
+   */
+   async listAmiUsage(conditions = {}, client = undefined) {
+    assert(typeof conditions === 'object');
```

Comment:
This assert should be moved to `this._listTable` or knocked out if it is already there.

Reply:
We also have that check in _listTable, but I've tended to over-assert in this codebase, so it's probably fine to leave this in here for consistency